frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/redux/stateUpdate/vmSettings/vm-settings-tab-state-update.ts

```diff
@@ -22,6 +23,38 @@ import { windowsToolsStorage } from '../../initial-state/storage-tab-initial-sta
 import { getStorages } from '../../../selectors/selectors';
 import { prefillVmTemplateUpdater } from './prefill-vm-template-state-update';
 
+export const selectUserTemplateOnLoadedUpdater = (options: UpdateOptions) => {
+  const { id, dispatch, getState } = options;
+  const state = getState();
+
+  if (
+    iGetCommonData(state, id, VMWizardProps.isCreateTemplate) ||
+    iGetVmSettingAttribute(state, id, VMSettingsField.USER_TEMPLATE, 'initialized')
```

Comment:
we should set the initialized property according to the `VMWizardProps.userTemplateName` and `VMWizardProps.isCreateTemplate` in vm-settings-tab-initial-state.ts

so we would not need this check here `VMWizardProps.isCreateTemplate`

Reply:
plus this should also break when there was no change to the templates

Reply:
about the initial state - shouldn't we set the value of template in the beggining so the user sees it? and maybe disable it for good for this wizard instance?

Reply:
> about the initial state - shouldn't we set the value of template in the beggining so the user sees it? and maybe disable it for good for this wizard instance?

done :cake:

Reply:
:birthday: :tada: fixed